app/views/prison/visits/_prisoner_details.html.erb

```diff
@@ -1,20 +1,20 @@
 <h2 class="bold-medium"><%= t('.title') %></h2>
 <% case @visit.prisoner_existance_status %>
-<% when StaffNomisChecker::VALID %>
+<% when PrisonerDetailsPresenter::VALID %>
```

Comment:
I think it would be better to consolidate this constant values across the classes so one doesn't have to know the particular namespace. Something like `Constants::VALID`, `Constants::INVALID`, etc.

What do you think?

Reply:
I thought about this too but I prefered to keep the constant close to were their values are used as a result of computation, makes it easier to navigate the code